.gitignore

```diff
@@ -1,6 +1,2 @@
-.idea
-.project
```

Comment:
These two should be ignored with a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).